src/color/p5.Color.js

```diff
@@ -446,52 +423,81 @@ var colorPatterns = {
  * </code>
  * </div>
  */
-p5.Color._getFormattedColor = function () {
+p5.Color._parseInputs = function() {
   var numArgs = arguments.length;
-  var mode    = this._renderer._colorMode;
-  var maxArr  = this._renderer._colorMaxes[this._renderer._colorMode];
+  var mode = this._renderer._colorMode;
+  var maxes = this._renderer._colorMaxes;
   var results = [];
```

Comment:
When I make this change it causes the `lerpColor()` HSB and HSL tests to fail, but I'm not sure why. It might be something to do with the way I’ve set up the tests. Any ideas? :confused: